tests/test_frame.py

```diff
@@ -2256,6 +2256,20 @@ def test_frame_stack() -> None:
             ),
             pd.Series,
         )
+        check(
+            assert_type(
+                df_multi_level_cols2.stack(0, future_stack=False),
+                Union[pd.DataFrame, "pd.Series[Any]"],
+            ),
+            pd.DataFrame,
+        )
```

Comment:
Can you surround this with `if PD_LTE_22:`  as the `future_stack` parameter is being removed in the future, and it will allow the nightly tests that get run upon a merge to succeed?

Reply:
Sure, I just added the condition 👍